### DISCUSSION TITLE
How to pass parameters between nested navigations

### DISCUSSION THREAD

**Discussion by joaoVictorDeAndrade** [react-navigation/react-navigation#10500] [General]
I have a Tab Navigator that renders two Tab Screens. (A and B).

Each Tab Screen has a component that is a Stack Navigator with Stack Screens.

How do I pass parameters between Tab Screen A to a Stack Screen that Stack Navigator calls?

**Comment by raajnadar** [comment]
Check this document for more info

https://reactnavigation.org/docs/nesting-navigators/#passing-params-to-a-screen-in-a-nested-navigator
